src/solver/conv_winoRxS.cpp

```diff
@@ -467,7 +476,7 @@ static bool IsApplicableBase(const ConvolutionContext& params)
         return false;
 
     // clang-format off
-    if (! ( (params.kernel_stride_w == 1 || params.kernel_stride_w == 2)
+    if (! (params.kernel_stride_w == 1
```

Comment:
Please check these changes. 
It means that the case is applicable only with `kernel_stride=1`, even if it was _F2x3_ solver.